### PR TITLE
some minor changes

### PR DESCRIPTION
**Pull request by olamy**
@oleg-nenashev  it looks I cannot push anymore directly....
so feel free to merge this.

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Will add you to developers

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 Changelog
 ===
 
+##4.2
+
+* Update Jetty from ``9.4.5.v20170502` to `9.4.8.v20171121`
```
Comment (oleg-nenashev) [inline CHANGELOG.md]: Please link changelogs

**Comment by oleg-nenashev** [comment]
Any change it fixes https://issues.jenkins-ci.org/browse/JENKINS-49596 ?

**Comment by oleg-nenashev** [comment]
We will likely need a second patch with backporting of session cleaning to 2.107.x.

**Comment by oleg-nenashev** [comment]
@olamy you should have write access to this repo now

**Comment by daniel-beck** [comment]
Lacks documentation for the new option.

**Comment by rodrigc** [comment]
@olamy See: https://issues.jenkins-ci.org/browse/JENKINS-33412?focusedCommentId=345986&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-345986

**Pull request merged**